bin/common/versions.sh

```diff
@@ -59,3 +59,11 @@ then
     echo ubuntu '       =' $UBUNTU_VERSION
     echo
 fi
+
+if id -u vagrant >& /dev/null; then
+  SCF_BIN_DIR="${SCF_BIN_DIR:-/usr/local/bin}"
+  SCF_TOOLS_DIR="${SCF_TOOLS_DIR:-/home/vagrant/tools}"
+else
+  SCF_BIN_DIR="${SCF_BIN_DIR:-output/bin}"
+  SCF_TOOLS_DIR="${SCF_TOOLS_DIR:-output/tools}"
```

Comment:
No. _The directories are not versions._  If you want to do this, do it in a file that's **not** named `versions.sh` please.

(I'd say to just set it in the Vagrantfile before running the install_tools scripts, and use the (already-existing) fallback in that script for the non-vagrant case.)